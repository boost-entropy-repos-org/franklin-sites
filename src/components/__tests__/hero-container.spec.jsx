import React from 'react';
import renderer from 'react-test-renderer';
import HeroContainer from '../hero-container';

describe('Hero Container component', () => {
  test('should render', () => {
    const component = renderer.create(
      <HeroContainer title="Title">
        <p>Body content</p>
      </HeroContainer>,
    );
    expect(component).toMatchSnapshot();
  });
});
