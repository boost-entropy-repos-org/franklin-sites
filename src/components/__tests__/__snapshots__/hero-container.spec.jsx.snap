// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Container component should render 1`] = `
<div
  className="hero-container"
>
  <h4>
    Title
  </h4>
  <p>
    Body content
  </p>
</div>
`;
